controllers/default.py

```diff
@@ -36,6 +36,11 @@ def user():
         @auth.requires_permission('read','table name',record_id)
     to decorate functions that need access control
     """
+    if request.args[0] == 'register':
+        response.title = 'iDibo Registration'
```

Comment:
Since the title of the page is something hardcoded, you could leave it in the static html page where it was before (in views -> default -> register.html -> <title> tag)